scss/templates/user.scss

```diff
@@ -204,6 +204,7 @@
 
   @include media-breakpoint-down(sm) {
     ul {
+      display: -ms-grid;
```

Comment:
do we need to put prefixes here? I think this should be handled by autoprefixer right?

Reply:
Yes we should, and this actually remind me to create the ticket. 
_I am having a extension checking upon a11y matters and it include some CSS rules depending upon supported browsers._

https://app.asana.com/0/1159164196409129/1199915290849534